lib/json_api/view.ex

```diff
@@ -0,0 +1,59 @@
+defmodule ElixirTools.JsonApi.View do
+  @moduledoc """
+  JSON API View is an overlay over the default Jaserializer view and provides functionality to apply
+  standards.
+  """
+
+  defmacro __using__(opts) do
+    quote do
+      use JaSerializer, unquote(opts)
+
+      def render("index.json-api", data) do
+        ElixirTools.JsonApi.View.render(__MODULE__, data)
+      end
+
+      def render("show.json-api", data) do
+        ElixirTools.JsonApi.View.render(__MODULE__, data)
+      end
+
+      def render("errors.json-api", data) do
+        JaSerializer.PhoenixView.render_errors(data)
+      end
+
+      def format_datetime(value) do
+        value
+        |> Timex.Timezone.convert("UTC")
+        |> DateTime.to_iso8601()
+      end
+    end
+  end
+
+  def render(serializer, data) do
+    serializer
+    |> JaSerializer.PhoenixView.render(data)
+    |> format()
+  end
+
+  @spec format(map) :: map
+  defp format(map) do
+    for {key, value} <- ensure_map(map), into: %{} do
+      key = convert_key(key)
```

Comment:
Opinions on variable rebinding? Do we use it often in our codebase? In my opinion it's a bit easier to debug when variable names are explicit, eg. `converted_key = convert_key(key)`. It's easier to keep track and follow the code.